lib/player_module.rb

```diff
@@ -0,0 +1,14 @@
+require "human"
+require "computer"
+
+module Player
+  def self.generate(type, board)
+    if type == 'hh'
```

Comment:
Constants for the Types?

Reply:
Also, something to think about as you proceed is that maybe you don't want PlayerFactory to know about display concerns like the game type.  What if you decide you want to change 'hh' to 'HH' or 'HVC', you then have to change PlayerFactory too.
Can you think of a way of having a GameType that is used by both your Display and your PlayerFactory?

Reply:
Thanks for this! Have mapped the input from display to the game type constants.